src/ProjectTemplates/BlazorWasm.ProjectTemplates/content/BlazorWasm-CSharp/Server/Data/WeatherForecastService.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BlazorWasm_CSharp.Server.Data
+{
+    public class WeatherForecastService
```

Comment:
Is this used anywhere? I can't see references to it. It seems unnecessary since `WeatherController` takes care of this an is simpler than having a separate "service" concept.

Reply:
Removed